java/base-client/src/main/java/no/bankaxept/epayment/client/base/accesstoken/ScheduledAccessTokenPublisher.java

```diff
@@ -23,27 +25,23 @@ public class ScheduledAccessTokenPublisher implements AccessTokenPublisher, Flow
 
     private final HttpClient httpClient;
     private final String uri;
-    private final LinkedHashMap<String, List<String>> headers;
+    private final Map<String, List<String>> headers;
+    private final String body;
 
     private final AtomicReference<AccessToken> atomicToken = new AtomicReference<>();
 
     private final Queue<Flow.Subscriber<? super String>> subscribers = new LinkedBlockingQueue<>();
 
-    public ScheduledAccessTokenPublisher(String uri, String apimKey, String username, String password, Clock clock, ScheduledExecutorService scheduler, HttpClient httpClient) {
+    private ScheduledAccessTokenPublisher(String uri, Map<String, List<String>> headers, String body, Clock clock, ScheduledExecutorService scheduler, HttpClient httpClient) {
```

Comment:
I changed this to a generic Map, I don't see why it needed to be a specific implementation? 🤔

Reply:
order of headers shouldn't matter